components/org.wso2.carbon.identity.application.authenticator.oidc/src/main/java/org/wso2/carbon/identity/application/authenticator/oidc/OpenIDConnectAuthenticator.java

```diff
@@ -309,6 +311,8 @@ protected String getUserInfoEndpoint(OAuthClientResponse token, Map<String, Stri
                     log.debug("Empty JSON response from user info endpoint. Unable to fetch user claims." +
                             " Proceeding without user claims");
                 }
+                diagnosticLog.info("Empty JSON response from user info endpoint. Unable to fetch user claims." +
```

Comment:
Can you check this is the actual case?
AFAIR we pick the claims from id_token if userinfo fails. So this log might be misleading :)

Reply:
fixed